lib/rubocop/cop/style/commented_keyword.rb

```diff
@@ -95,6 +98,13 @@ def source_line(comment)
           comment.source_range.source_line
         end
 
+        def steep_annotation?(line, comment)
```

Comment:
This is a minor point, `steep_annotation?` is used later than `rbs_inline_annotation?`. Therefore, it would be more logical to definite it after `def rbs_inline_annotation?`.

Reply:
@koic 

I agree with your opinion.
I think it's important about logical relationships, especially since code is read more than written.
I fixed it. Thanks!

https://github.com/rubocop/rubocop/pull/13943/commits/583b834a2d0b13b2e521986cee9b21a371bc934c#diff-33530142aa1144a74752b8678c37483180c7f32218b5503e08997ab076823b91R112-R117